lib/Installation/Popups/LicensePopup.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles License Popup.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Installation::Popups::LicensePopup;
+use parent 'YaST::PageBase';
+use strict;
+use warnings;
+use Installation::Popups::AcceptPopup;
+use YuiRestClient;
+
+sub init {
+    my $self = shift;
+    $self->{popup_accept} = Installation::Popups::AcceptPopup->new({app => YuiRestClient::get_app()});
+    $self->{rct_license} = $self->{app}->richtext({id => "lic"});
+    return $self;
+}
+
+sub accept_license_popup {
+    my ($self) = @_;
+    $self->{popup_accept}->press_accept();
```

Comment:
now we don't use controller for this, so you need to call the getter before each call:
`$self->get_license_popup()->{popup_accept}->press_accept();` to know that the element is really there when you click it.
To distinguish methods that belong to the interface, (public accesible from outside) we can name the private one with underscore, like in this case `_get_license_popup`.

Reply:
done